charts/streamsets/control-agent/2.0.101/Chart.yaml

```diff
@@ -9,6 +10,7 @@ keywords:
 - streamsets
 - sdc
 - sch
+kubeVersion: 1.17 - 1.21
```

Comment:
index.yaml should be modified https://github.com/rancher/partner-charts/blob/6f3b252e5c217636bcd575627fb7aca0b3ac4abd/index.yaml#L456-L477

Can you delete the above entry and run `make charts` to regenerate it?

Reply:
Done